src/index.ts

```diff
@@ -1,5 +1,25 @@
 import * as _ from 'lodash';
+import * as THREE from 'three';
+import cube from './Cubes';
+import scene from './Scene';
+import camera from './Camera';
+import renderer from './Renderer';
 
+scene.add(cube);
+camera.position.set(0, 0, 10);
+camera.lookAt(0,0,0); 
+
+//animation frame for cube
+function animate() {
+  requestAnimationFrame( animate );
+
+  cube.rotation.x += 0.01;
```

Comment:
Nice work on using `requestAnimationFrame`!
Something to consider later is to calculate the time between these calls and use it to animate, so that the rotation speed is unaffected by the rendering speed. This is future work so there is no need to implement it right now.